### PR TITLE
add facility modal made consistent with edit facility

### LINKED ISSUE #3075: Make the Add Facility modal and Edit Facility modal consistent
### Describe the bug

Make the Add Facility modal and Edit Facility modal responsive and consistent.

### PR DESCRIPTION
**Pull request by skshahnawaz**
Fixes #3075 

#### Short description of what this resolves:
The overall form presentation of "add facility" has been made consistent with "edit facility"

#### Changes proposed in this pull request:

- The arrangement of the form fields has been altered in "add facility" and made consistent with "edit facility"
- "Add facility" modal has been made responsive

Screenshot 2020-03-05 at 4 29 33 PM
Screenshot 2020-03-05 at 4 29 45 PM
Screenshot 2020-03-05 at 4 29 37 PM

### CODE REVIEW

**Comment by skshahnawaz** [comment]
@bradymiller Please review the changes

**Comment by mdsupport** [comment]
@skshahnawaz If you are working on this form, you should consider replacing alerts by bootstrap alerts.  In general popups interrupt the flow but JS alerts in a popup make it worse.  Maybe @tywrenn has some standards in mind?

**Comment by sjpadgett** [comment]
> replacing alerts by bootstrap alerts.

Check out library/js/utility.js or library/dialog.js. Both have timed bs alerts based on promise. Plus js translation utility. Usually call with an await. I have a couple examples in code if interested

**Comment by skshahnawaz** [comment]
> > replacing alerts by bootstrap alerts.
> 
> Check out library/js/utility.js or library/dialog.js. Both have timed bs alerts based on promise. Plus js translation utility. Usually call with an await. I have a couple examples in code if interested

yes please share the examples so that I get an idea about the same

**Comment by skshahnawaz** [comment]
Can I create a separate issue for fixing the alerts?

**Comment by sjpadgett** [comment]
> yes please share the examples so that I get an idea about the same

Just checkout utility.js. Has example use.
/*
* function syncAlertMsg(()
*
* Universal sync BS alert message returns promise after resolve.
* Call below to return a promise after alert is resolved.
* Example: syncAlertMsg('Hello, longtime, 'success', 'lg').then( asyncRtn => ( ... log something });
*
* Or use as IIFE to run inline.
* Example:
*   (async (time) => {
*       await asyncAlertMsg('Waiting till x'ed out or timeout!', time); ...now go;
*   })(3000).then(rtn => { ... but then could be more });
*
* */
async function syncAlertMsg(message, timer = 5000, type = 'danger', size = '') {
    return await asyncAlertMsg(message, timer, type, size);
}



> Can I create a separate issue for fixing the alerts?

Sure, thanks.

**Comment by skshahnawaz** [comment]
> > yes please share the examples so that I get an idea about the same
> 
> Just checkout utility.js. Has example use.
> /*
> 
> * function syncAlertMsg(()
> * Universal sync BS alert message returns promise after resolve.
> * Call below to return a promise after alert is resolved.
> * Example: syncAlertMsg('Hello, longtime, 'success', 'lg').then( asyncRtn => ( ... log something });
> * Or use as IIFE to run inline.
> * Example:
> * (async (time) => {
> * ```
>     await asyncAlertMsg('Waiting till x'ed out or timeout!', time); ...now go;
>   ```
> * })(3000).then(rtn => { ... but then could be more });
> * */
>   async function syncAlertMsg(message, timer = 5000, type = 'danger', size = '') {
>   return await asyncAlertMsg(message, timer, type, size);
>   }
> 
> > Can I create a separate issue for fixing the alerts?
> 
> Sure, thanks.

Please review and merge the current PR @sjpadgett . I'll close this issue and create a separate issue and start working on that

File: interface/usergroup/facilities_add.php [hunk 1]
```diff
@@ -184,7 +184,7 @@ function displayAlert() {
             <input type="hidden" name="mode" value="facility" />
 
             <div class="row">
-                <div class="col-sm-6">
+                <div class="col-6">
```
Comment (sjpadgett) [inline interface/usergroup/facilities_add.php]: Not sure why we need to restrict to col-6 inside a dialog. Especially for a vertical form. Forms in dialogs should be 100% width where possible as they tend to behave better in responsive and will sixe to the calling dialog ie looks like a lot of padding. This keeps from sizing the dialog to the form. IMO, mind you.
Just leave though for now and ketch on BS alert issue which I think you'll find fun to do:)

File: interface/usergroup/facilities_add.php [hunk 2]
```diff
@@ -273,62 +314,27 @@ function displayAlert() {
...
-                <label for="oid"><?php echo xlt('OID'); ?>: </label>
-                <input class="form-control" type="entry" size="20" name="oid" value="<?php echo attr($facility["oid"]) ?>" />
-            </div>
+            
+            
+            
+            
             <div class="form-group">
                 <label for="mail_stret"><?php echo xlt('Mailing Address'); ?>: </label>
                 <input class="form-control" type="entry" size="20" name="mail_street" value="<?php echo attr($facility["mail_street"]) ?>" />
```
Comment (sjpadgett) [inline interface/usergroup/facilities_add.php]: label for doesn't have a counterpart.

**Comment by sjpadgett** [comment]
@skshahnawaz I don't have time to test right now but will merge so you can work on alert issue.
Open issue and i'll assign and thanks for the work here...

**Comment by sjpadgett** [comment]
@bradymiller This is another PR where admin was given authorship. I'm not sure what is different with these guys repo's that may cause this. Would you fix for me sir?

**Comment by bradymiller** [comment]
Sure thing. Will  fix it tonight. I think the issue is recent github
changes (hopefully a bug on their end and not purposeful behavior) and not
because of individual repos; still looking into it.

**Comment by bradymiller** [comment]
@sjpadgett , I just fixed authorship in codebase by reverting original commit and bringing in same commit with correct author.

**Pull request merged**